hips/utils/wcs.py

```diff
@@ -4,16 +4,20 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
+from astropy.coordinates import Angle
 from astropy.wcs.utils import pixel_to_skycoord, wcs_to_celestial_frame
 
 __all__ = [
     'WCSGeometry',
 ]
 
-__doctest_skip__ = ['WCSGeometry']
+__doctest_skip__ = [
+    'WCSGeometry',
+    'WCSGeometry.*'
```

Comment:
Why is this second entry `WCSGeometry.*` needed here?
Shoudn't the first `WCSGeometry` entry lead to all doctests in this class being skipped?
Please consult the Astropy docs on this point if you're not sure.